doc/cygnus-ngsi/installation_and_administration_guide/README.md

```diff
@@ -12,12 +12,14 @@
 * Running:
     * [Running as a process](./running_as_process.md)
     * [Running as a service](./running_as_service.md)
-* Advanced features:
+* Advanced topics:
     * [Testing](./testing.md)
     * [Name mappings](./name_mappings.md)
     * [Grouping Rules](./grouping_rules.md)
     * [Multitenancy](./multitenancy.md)
     * [Performance tips](./performance_tips.md)
+    * [NGSIv2 support](./ngsiv2_support.md)
+    * [IPv6 support](./ipv6_support.md)
```

Comment:
ipv6_support.md is not part of this PR... I understand this is just a fix in the RTD index and that .md was added in a previous PR, isn't it?